<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormulasTab</class>
 <widget class="QWidget" name="FormulasTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>588</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_3">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QSplitter" name="splitter_2">
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTreeWidget" name="FormulaTree">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Tree of all formulas</string>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string notr="true">Formulas</string>
        </property>
       </column>
      </widget>
      <widget class="QScrollArea" name="FormulaScrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>List of all formulas</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>375</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QRadioButton" name="FormulaSolveSolveSet">
           <property name="toolTip">
            <string>See SymPy 'Solveset' for more information</string>
           </property>
           <property name="text">
            <string>Solveset</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="FormulaSolveSolve">
           <property name="toolTip">
            <string>Algebraically solves equation</string>
           </property>
           <property name="text">
            <string>Solve</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QPushButton" name="FormulaPreview">
           <property name="toolTip">
            <string>Preview the Equation</string>
           </property>
           <property name="text">
            <string>Preview</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QPushButton" name="FormulaCalculate">
           <property name="toolTip">
            <string>Calculate the Equation - Shift + Enter</string>
           </property>
           <property name="text">
            <string>Calculate</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="childrenCollapsible">
            <bool>false</bool>
           </property>
           <widget class="QTextBrowser" name="FormulaExact">
            <property name="toolTip">
             <string>Exact answer of calculation</string>
            </property>
            <property name="lineWrapMode">
             <enum>QTextEdit::NoWrap</enum>
            </property>
            <property name="placeholderText">
             <string>Exact Answer</string>
            </property>
           </widget>
           <widget class="QTextBrowser" name="FormulaApprox">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Approximate answer of calculation</string>
            </property>
            <property name="placeholderText">
             <string>Approximate Answer</string>
            </property>
           </widget>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Domain of equation, only works on Solveset unless 'verify domain' is checked</string>
             </property>
             <property name="text">
              <string>Domain</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="FormulaDomain">
             <property name="toolTip">
              <string>Domain of equation, only works on Solveset unless 'verify domain' is checked</string>
             </property>
             <property name="currentIndex">
              <number>4</number>
             </property>
             <item>
              <property name="text">
               <string>Naturals</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Naturals0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Integers</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Reals</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Complexes</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Interval(-oo, 0)</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="FormulaNsolve">
           <property name="toolTip">
            <string>Approximate equation using nsolve()</string>
           </property>
           <property name="text">
            <string>Approximate</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="FormulaStartV">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Starting point that's close to the solution</string>
           </property>
           <property name="placeholderText">
            <string>Starting Vector (x0)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
